Documentation/operations/troubleshooting.rst

```diff
@@ -537,6 +537,93 @@ Opening the ``status``, we can drill down through ``policy.realized.l4``. Do
+                                                                                                                                                                                                                16860295
+                                                                                                                                                                                                                16860322
+                                                                                                                                                                                                                16860323
+                                                                                                                                                                                                                16860324
+                                                                                                                                                                                                                16860325
+                                                                                                                                                                                                                16860326
+                                                                                                                                                                                                                16860327
+                                                                                                                                                                                                                16860328
+    &LabelSelector{MatchLabels:map[string]string{any.name: netperf,k8s.io.kubernetes.pod.namespace: default,},MatchExpressions:[]LabelSelectorRequirement{},}           default/tofqdn-dns-visibility   1
+    &LabelSelector{MatchLabels:map[string]string{cidr.1.1.1.1/32: ,},MatchExpressions:[]LabelSelectorRequirement{},}                                                    default/tofqdn-dns-visibility   1       16860329
+    &LabelSelector{MatchLabels:map[string]string{cidr.1.1.1.2/32: ,},MatchExpressions:[]LabelSelectorRequirement{},}                                                    default/tofqdn-dns-visibility   1       16860330
+    &LabelSelector{MatchLabels:map[string]string{cidr.1.1.1.3/32: ,},MatchExpressions:[]LabelSelectorRequirement{},}                                                    default/tofqdn-dns-visibility   1       16860331
+
+From the output above, we see that all 3 selectors are in use. The significant
+action here is to determine which selector is selecting the most identities,
+because the policy containing that selectors is the likely cause for the
+policymap pressure.
```

Comment:
```suggestion
From the output above, we see that all three selectors are in use. The significant
action here is to determine which selector is selecting the most identities,
because the policy containing that selector is the likely cause for the
policymap pressure.
```